main.py

```diff
@@ -10,6 +10,8 @@
 from src.corenodes.display import InputModule, OutputModule
 from src.corenodes.transform import BlurModule, ResizeModule, RotateModule
 
+DEV_MODE = True
+
```

Comment:
Would be nice that instead of using a variable to check whether to display the debug window, we could instead have a shortcut like F12 that opens the debug window. It should also be toggleable so pressing F12 again would close it.